docs/3.0/common/associations_attach_scope_option_common.md

```diff
@@ -13,10 +13,23 @@ field :user,
   attach_scope: -> { query.non_admins }
 ```
 
+
 Pass in a block where you attach scopes to the `query` object and `parent` object, which is the actual record where you want to assign the association. The block is executed in the [`ExecutionContext`](./../execution-context).
 :::
 
 :::warning
 The `attach_scope` will not filter the records in the listing from `has_many` or `has_and_belongs_to_many` associations.
 Use [`scope`](#scope) or a [Pundit policy `Scope`](./../authorization#scopes) for that.
+
+Pass in a block where you attach scopes to the `query` object. The block is executed in the [`ExecutionContext`](./../execution-context).
```

Comment:
I don't understand this sentence. I mean, what should the user take from it?

Reply:
My mistake. It was something I copied from above and forgot to delete.